packages/affiliates/test/mocha/affiliates.js

```diff
@@ -51,10 +55,10 @@ describe("DeployerRewards", function() {
       assert.ok(history.history.length());
     });
   });
-  it("getCollateralPriceHistory", async function() {
+  it("getCoingeckoPriceHistory", async function() {
     this.timeout(10000);
     const [, address] = collateralTokens;
-    const result = await affiliates.utils.getCollateralPriceHistory(address, "usd", startingTimestamp, endingTimestamp);
+    const result = await affiliates.utils.getCoingeckoPriceHistory(address, "usd", startingTimestamp, endingTimestamp);
```

Comment:
Is this external API call potentially problematic to put into our CI?

Reply:
I dont think this was problematic, though it does reach outside, so it technically could be. i think the problem was with the getSyntheticPrice which was causing an issue since it relied on web3 mainnet and abis.